pyvista/plotting/renderer.py

```diff
@@ -2891,6 +2891,13 @@ def close(self):
             self._empty_str.SetReferenceCount(0)
             self._empty_str = None
 
+    def render(self):
+        """Notify renderer components of manual render call."""
+        if self.__charts is not None:
+            for chart in self.__charts:
+                # Notify ChartMPLs to redraw themselves when plotter.render() is called
+                chart._render_event(plotter_render=True)
```

Comment:
Hm, I appreciate you changing this to `render()` but now I'm thinking`self.Render()` should be called here, i.e., `vtk.vtkRenderer.Render`

Reply:
Or if you want to avoid the overhead of calling `Render()` then changing this method to `render_charts()` might be a good option

Reply:
Calling `self.Render()` would lead to duplicate render calls though, so I don't think we should do that. I don't really like the current naming either to be honest, but I couldn't come up with a good alternative yet, suggestions are welcome.
The purpose of the method is to notify charts of an upcoming render event that is manually triggered (by an explicit `plotter.update` or `plotter.render` call), so maybe `on_plotter_render` or `on_explicit_render` (these are quite long), or `notify_render`, `on_render` or `before_render` (the problem I have with these is that the method is not called for all render events which might be confusing)?

Reply:
I went ahead with `on_plotter_render` as the method name.